packages/angular/projects/ui-angular/src/lib/components/authenticator/components/authenticator/authenticator.component.html

```diff
@@ -1,22 +1,33 @@
 <div data-amplify-authenticator [attr.data-variation]="variation">
   <div data-amplify-container>
     <amplify-slot name="header" [context]="context"></amplify-slot>
-    <div data-amplify-router>
+    <div
+      data-amplify-router
+      [attr.data-amplify-router-content]="hasTabs() ? undefined : ''"
```

Comment:
For screens with tabs, the actual scrollable content (ie. `data-amplify-router-content`) is the `tab-item` inside this container. For tab-less screens, this distinction is unneeded and `data-amplify-router-content` is safely placed on this container. See attached picture in the PR.

Reply:
How will this work for customers who hide the tabs with CSS?

Reply:
`data-amplify-router` and `data-amplify-router-content` styles do not affect tab headers, so hiding tabs with CSS is unaffected by this change.